src/styles/box/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  justify-content: center;
+  flex-flow: column nowrap;
+  align-items: center;
+
+  &__pictures {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    &-name {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      text-align: left;
+      color: #060b35;
+    }
+
+    &-code {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: left;
+      color: #616070;
+    }
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+
+    &-rewiews {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: right;
+      padding-left: 17px;
+      align-self: center;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    &-let {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      color: #616070;
+      padding-right: 85px;
+    }
+
+    &-num {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    cursor: pointer;
+    background-color: $button-color;
+    outline: none;
+    width: 166px;
+    height: 40px;
+    border: none;
+    border-radius: 5px;
+
+    &-text {
+      color: #fff;
+    }
+
+    &:hover {
+      background-color: #fff;
+      color: $button-color;
+      border: 1px solid $button-color;
+
+      &-text {
```

Comment:
If we write it like a normal class it will be card__button:hovercard__button:hover-text

& copied the name of the class that wraps